src/wp-includes/blocks/template-part.php

```diff
@@ -276,7 +276,7 @@ function register_block_core_template_part() {
 		__DIR__ . '/template-part',
 		array(
 			'render_callback' => 'render_block_core_template_part',
-			'variations'      => build_template_part_block_variations(),
+			'variations'      => 'build_template_part_block_variations',
```

Comment:
I think it's okay to have these changes here for now and remove them before merging into the core. Then, we can update the code in the `block-library` package in a backward compatible way. The Gutenberg plugin supports multiple WP versions.